pkgs/development/compilers/llvm/common/compiler-rt/default.nix

```diff
@@ -149,7 +150,9 @@ stdenv.mkDerivation ({
     ln -s $out/lib/*/clang_rt.crtbegin_shared-*.o $out/lib/crtbeginS.o
     ln -s $out/lib/*/clang_rt.crtend_shared-*.o $out/lib/crtendS.o
   '' + lib.optionalString doFakeLibgcc ''
-     ln -s $out/lib/freebsd/libclang_rt.builtins-*.a $out/lib/libgcc.a
+     ln -s $out/lib/*/libclang_rt.builtins-*.a $out/lib/libgcc.a
+  '' + lib.optionalString forceLinkCompilerRt ''
+     ln -s $out/lib/*/libclang_rt.builtins-*.a $out/lib/libcompiler_rt.a
```

Comment:
Any idea why we need this?

Reply:
I think the build system simply doesn't bother to make the "normally named" library when its only statically linking? I am a bit confused why too. But note how the previous hack also had to work from the more obscure `libclang_rt.builtins`.

Reply:
I suppose we can add a comment explaining this

Reply:
Added those comments